app/core/edge_inference.py

```diff
@@ -172,10 +174,7 @@ def detector_configured_for_local_inference(self, detector_id: str) -> bool:
         Returns:
             True if the detector is configured to run local inference, False otherwise
         """
-        if not self.inference_configs:
-            return False
-
-        return detector_id in self.inference_configs.keys() and self.inference_configs[detector_id].enabled
+        return detector_id in self.edge_config.detectors.keys() and self.edge_config.detectors[detector_id].enabled
```

Comment:
Something interesting I noticed is that the `enabled` option doesn't actually prevent an inference pod from being created for a detector when False, it just prevents edge inference from occurring. I created a separate Jira ticket to address this.